docs/language/accounts.mdx

```diff
@@ -228,6 +228,24 @@ transaction(publicKey: [UInt8]) {
 }
 ```
 
+To add a public key to the current auth account:
```

Comment:
Maybe clarify what "current" means, e.g.:

```suggestion
To add a public key to an existing account, which signed the transaction:
```

Reply:
Love it, thanks!